redun/tests/test_scheduler.py

```diff
@@ -78,6 +78,19 @@ def workflow():
 
     assert scheduler.run(workflow()) == "hello world"
 
+    @task()
```

Comment:
These are good tests. Could you move them to `test_default_args` in this file? I think that might be a more appropriate test. Thanks!